plugin/virustotal/vt_ida/plugin_loader.py

```diff
@@ -523,20 +540,24 @@ def init(self):
       self.menu = Popups()
       self.menu.hook()
       arch_info = idaapi.get_inf_structure()
+      if idaapi.IDA_SDK_VERSION >= 770:
```

Comment:
I'd do this code instead, so that the code works without needing to know the specific version of the SDK.

Also proposing a standalone function so that it is more clear...

class Error(Exception):
  pass

class IncompatibleIdaVersion(Error):
  pass

def get_procname(arch_info):
  try:
    return arch_info.procname
  except AttributError:
    pass
  try:
    return arch_info.procName
  except AttributError:
    pass
  # IDA has changed the API, complain loudly and raise an exception.
  if procname is None:
    logging.error('Could')
    raise IncompatibleIdaVersion('Could not get procname from arch_info.')

Reply:
Sorry for the typo... should be "AttributeError"!!